testutil/testutil.go

```diff
@@ -0,0 +1,58 @@
+package testutil
```

Comment:
In this file, can you put a comment above `package testutil` starting with `// package testutil` so that godoc has something to say about the package in general?